### ISSUE TITLE
WildOnCam searching CherryPimps instead of WildOnCam

### ISSUE THREAD

**Issue by covrdinbeez** [PAhelper/PhoenixAdult.bundle#925]
Looks like WildOnCam (which is a subsite of CherryPimps) is sending search queries to CherryPimps instead of WildOnCam's site. Doesn't look like results for WildOnCam are always showing on the CherryPimps site.

Example:
Search: "WildOnCam Violet Rain"
Log:
```
2020-08-04 14:45:55,269 (700003114000) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM2CjcwNTUzMnMyCmlkczIxCldpbGRPbkNhbSBWaW9sZXQgUmFpbnM0Cm5hbWVzNAoyMDE4czQKeWVhcnIwCg__
2020-08-04 14:45:55,270 (700003114000) :  DEBUG (runtime:49) - Received packed state data (80 bytes)
2020-08-04 14:45:55,271 (700003114000) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM2CjcwNTUzMnMyCmlkczIxCldpbGRPbkNhbSBWaW9sZXQgUmFpbnM0Cm5hbWVzNAoyMDE4czQKeWVhcnIwCg__
2020-08-04 14:45:55,271 (700003114000) :  INFO (agentkit:961) - Searching for matches for {'year': '2018', 'id': '705532', 'name': 'WildOnCam Violet Rain'}
2020-08-04 14:45:55,271 (700003114000) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/library/metadata/705532/tree'
2020-08-04 14:45:55,274 (700003114000) :  INFO (__init__:802) - *******MEDIA TITLE****** WildOnCam Violet Rain
2020-08-04 14:45:55,274 (700003114000) :  INFO (__init__:802) - Getting Search Settings for: WildOnCam Violet Rain
2020-08-04 14:45:55,274 (700003114000) :  INFO (PAsearchSites:1106) - mediaTitle w/ possible abbreviation: WildOnCam Violet Rain
2020-08-04 14:45:55,279 (700003114000) :  INFO (PAsearchSites:1361) - mediaTitle w/ possible abbrieviation fixed: WildOnCam Violet Rain
2020-08-04 14:45:55,281 (700003114000) :  INFO (PAsearchSites:1058) - Site found with method #3
2020-08-04 14:45:55,281 (700003114000) :  INFO (PAsearchSites:1361) - ^^^^^^^ siteID: 784
2020-08-04 14:45:55,281 (700003114000) :  INFO (PAsearchSites:1361) - ^^^^^^^ Shortening Title
2020-08-04 14:45:55,281 (700003114000) :  INFO (PAsearchSites:1361) - wildoncam v vs wild on cam
2020-08-04 14:45:55,281 (700003114000) :  INFO (PAsearchSites:1366) - 3
2020-08-04 14:45:55,281 (700003114000) :  INFO (PAsearchSites:1373) - 8
2020-08-04 14:45:55,282 (700003114000) :  INFO (PAsearchSites:1385) - searchTitle (before date processing): Violet Rain
2020-08-04 14:45:55,282 (700003114000) :  INFO (__init__:802) - Search Title: Violet Rain
2020-08-04 14:45:55,282 (700003114000) :  INFO (__init__:802) - Violet%20Rain
2020-08-04 14:45:55,283 (700003114000) :  INFO (PAutils:102) - Requesting GET "https://cherrypimps.com/search.php?query=Violet+Rain&page=1"
2020-08-04 14:45:56,135 (700003114000) :  INFO (PAutils:102) - Requesting GET "https://cherrypimps.com/search.php?query=Violet+Rain&page=2"
2020-08-04 14:45:56,705 (700003114000) :  DEBUG (runtime:88) - Sending packed state data (112 bytes)
2020-08-04 14:45:56,705 (700003114000) :  DEBUG (runtime:924) - Response: [200] str, 320 bytes
```

This returns no results, as does the same search by hand on the CherryPimps site:
https://cherrypimps.com/search.php?x=0&y=0&query=Violet+Rain

But if I search on WildOnCam, I get the result I'm looking for:
https://wildoncam.com/search.php?x=0&y=0&query=Violet+Rain

And, as you can see (and I can confirm from a visual comparison) it looks like the structure of the sites is identical, just not the data.

Looks like we just need the term WildOnCam to return results from the WildOnCam site instead of CherryPimps.

**Comment by DirtyRacer1337** [comment]
Funny, they have that scene
https://cherrypimps.com/trailers/16476-violetrain.html

**Comment by covrdinbeez** [comment]
Looks like I've found another one: Izzy Lush.
https://wildoncam.com/search.php?query=Izzy+Lush&page=1
https://cherrypimps.com/search.php?query=Izzy+Lush&page=1

Specifically I'm trying to match the WildOnCam scene "Naughty Girl Izzy Lush Teasing You LIVE." It shows in the WildOnCam search above, but not through CherryPimps. Also, if you look at the two searches above, you'll see that they are showing different scenes for the same search.

Can we please get the WildOnCam search to actually search the WildOnCam site, not just search CherryPimps? Thanks.

**Comment by DirtyRacer1337** [comment]
Fixed in 2fb7b07eac0a28210b0716747bbbb6f5362156ea
